### PR TITLE
NXP-21383: prevent random bug failure

### PR DESCRIPTION
**Pull request by mincong-h**
This PR aims to prevent random failure like https://qa.nuxeo.org/jenkins/job/master/job/addons_FT_nuxeo-agenda-master/580/

### CODE REVIEW

File: ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java [hunk 1]
```diff
@@ -82,14 +82,14 @@ protected DocumentBasePage createTestEvent(DocumentBasePage page) throws IOExcep
     }
 
     @Test
+    @Repeat(issue = "NXP-21383: The text of 'betweenBanner' cannot be found before timeout.")
```
Comment (guirenard) [inline ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java]: Do not use @Repeat for IT tests, better fix test with the waitUntil functions from Locator

File: ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2012 Nuxeo SA (http://nuxeo.com/) and contributors.
+ * (C) Copyright 2012-2017 Nuxeo SA (http://nuxeo.com/) and contributors.
```
Comment (guirenard) [inline ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java]: Just 2017 is enough ;)

**Review by guirenard — CHANGES_REQUESTED** [review]
Just try first with Locator.waitForTextPresent(By.id("betweenBanner"), "Incoming Events");

File: ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java (hunk 1 above)
Comment (mincong-h) [reply ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java]: > Do not use `@Repeat` for IT tests

I agree with you.

> better fix test with the waitUntil functions from Locator

In this commit, the method `Locator#waitForTextPresent` is used and it already waits for 30 seconds. So just removing the `@Repeat` annotation would be enough, or other "wait-until" function(s) should be added to ensure the robustness?

Comment (guirenard) [reply ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java]: yes just removing @Repeat is ok

Comment (mincong-h) [reply ftest/webdriver/src/test/java/com/nuxeo/functionaltests/ITEventCreationTest.java]: Done, see commit 7321908.

**Comment by guirenard** [comment]
:+1:

**Review by guirenard — APPROVED** [review]

**Comment by mincong-h** [comment]
Merged. Thanks for your review, @guirenard.

**Pull request merged**